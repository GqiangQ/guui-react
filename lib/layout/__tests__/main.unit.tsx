import {configure} from 'enzyme';
import * as Adapter from 'enzyme-adapter-react-16';
import * as React from 'react';
import * as renderer from 'react-test-renderer';
import Main from '../main';

configure({adapter: new Adapter()});

describe(Main.displayName, () => {
  describe('props: className', () => {
    it('= xxx', () => {
      const tree = renderer
        .create(<Main className="xxx"></Main>)
        .toJSON();
      expect(tree).toMatchSnapshot();
    });
  });
  describe('props: style', () => {
    it(`= {color: 'red'}`, () => {
      const tree = renderer
        .create(<Main style={{color: 'red'}}></Main>)
        .toJSON();
      expect(tree).toMatchSnapshot();
    });
  });
});
