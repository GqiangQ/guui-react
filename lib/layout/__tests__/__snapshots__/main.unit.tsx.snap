// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main props: className = xxx 1`] = `
<div
  className="gu-main xxx"
/>
`;

exports[`Main props: style = {color: 'red'} 1`] = `
<div
  className="gu-main "
  style={
    Object {
      "color": "red",
    }
  }
/>
`;
